### ISSUE TITLE
Incompatibility with Html-webpack-plugin >=4

### ISSUE THREAD

**Issue by agavrilov** [OpenNTF/dojo-webpack-plugin#290]
Webpack compilation fails if I include Dojo webpack plugin and Html webpack plugin (version >=4) in the same build. 

See https://github.com/agavrilov/dojo-webpack-plugin-sample/tree/html-webpack-plugin-error for an example code (note: it is in '**html-webpack-plugin-error**' branch).

The build error thrown:
```
ERROR in   Error: C:\P\git\dojo-webpack-plugin-sample\template.html:177
  /******/  throw new Error('Module not found: ' + mid);
            ^
  Error: Module not found: C:\P\git\dojo-webpack-plugin-sample\node_modules\lodash\lodash.js

  - template.html:177 findModule
    C:/P/git/dojo-webpack-plugin-sample/template.html:177:21

  - template.html:203 contextRequire
    C:/P/git/dojo-webpack-plugin-sample/template.html:203:23

  - template.html:100 req
    C:/P/git/dojo-webpack-plugin-sample/template.html:100:21

  - template.html:297 Object.<anonymous>
    C:/P/git/dojo-webpack-plugin-sample/template.html:297:9

  - template.html:21 __webpack_require__
    C:/P/git/dojo-webpack-plugin-sample/template.html:21:30

  - template.html:284
    C:/P/git/dojo-webpack-plugin-sample/template.html:284:18

  - template.html:287
    C:/P/git/dojo-webpack-plugin-sample/template.html:287:10

  - index.js:314 HtmlWebpackPlugin.evaluateCompilationResult
    [dojo-webpack-plugin-sample]/[html-webpack-plugin]/index.js:314:28

  - index.js:230
    [dojo-webpack-plugin-sample]/[html-webpack-plugin]/index.js:230:22
```

**Comment by chuckdumont** [comment]
Thanks for the test case.  Investigating...

**Comment by chuckdumont** [comment]
@agavrilov - Please try release 2.8.16-rc1 and verify that it resolves your issue.

**Comment by agavrilov** [comment]
@chuckdumont  We are also using 'mini-css-extract-plugin'. It looks like it is causing the similar issue. See my updated branch at https://github.com/agavrilov/dojo-webpack-plugin-sample/tree/html-webpack-plugin-error
```
ERROR in   Error: C:\P\git\dojo-webpack-plugin-sample\node_modules\css-loader\dist\cjs.js!C:\P\git\dojo-webpack-plugin-sample\node_modules\dijit\themes\claro\  claro.css:168
  /******/  throw new Error('Module not found: ' + mid);
            ^
  Error: Module not found: C:\P\git\dojo-webpack-plugin-sample\node_modules\lodash\lodash.js

  - claro.css:168 findModule
    [dojo-webpack-plugin-sample]/[dojo-webpack-plugin-sample]/[dijit]/themes/claro/claro.css:168:21

  - claro.css:194 contextRequire
    [dojo-webpack-plugin-sample]/[dojo-webpack-plugin-sample]/[dijit]/themes/claro/claro.css:194:23

  - claro.css:100 req
    [dojo-webpack-plugin-sample]/[dojo-webpack-plugin-sample]/[dijit]/themes/claro/claro.css:100:21

  - template.html:98 Object.<anonymous>
    C:/P/git/dojo-webpack-plugin-sample/template.html:98:9

  - template.html:21 __webpack_require__
    C:/P/git/dojo-webpack-plugin-sample/template.html:21:30

  - template.html:85
    C:/P/git/dojo-webpack-plugin-sample/template.html:85:18

  - template.html:88
    C:/P/git/dojo-webpack-plugin-sample/template.html:88:10

  - index.js:314 HtmlWebpackPlugin.evaluateCompilationResult
    [dojo-webpack-plugin-sample]/[html-webpack-plugin]/index.js:314:28

  - index.js:230
    [dojo-webpack-plugin-sample]/[html-webpack-plugin]/index.js:230:22

  - async Promise.all

  - async Promise.all

```

**Comment by agavrilov** [comment]
@chuckdumont  Unfortunately, I cannot re-open this issue. Should I open a new one?

**Comment by chuckdumont** [comment]
Try adding the following to the DojoWebpackPlugin options in your webpack.config.js:

```javascript
	ignoredCompilationNames: [/^mini-css-extract-plugin\s/],
```

**Comment by agavrilov** [comment]
@chuckdumont  It works! Thanks!

**Comment by agavrilov** [comment]
@chuckdumont Is there any plan to release a new version with this fix? Thanks!

**Comment by chuckdumont** [comment]
I've published v2.8.16.  Note that you no longer need the `ignoreCompilationNames` entry in the dojo-webpack-plugin options, however, it won't cause any problems leaving it there.
